x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -122,11 +122,6 @@ func runCmd(
 	cmd.Env = append(os.Environ(), "NODE_ENV=production")
 	cmd.Args = append(cmd.Args, "--rich-events")
 
-	if len(params) > 0 {
```

Comment:
I moved this lower since that we we only have to instantiate `loggableCmd` once without updating args as they are added.